web/src/app/services/job/job.service.ts

```diff
@@ -29,11 +29,32 @@ import {SurveyService} from 'app/services/survey/survey.service';
   providedIn: 'root',
 })
 export class JobService {
+  defaultColors = List([
+    '#F37C22',
```

Comment:
Please refactor these valiues in appropriately named constants (RED, BLUE, YELLOW).